### ISSUE TITLE
Default carousel template is not accessible 

### ISSUE THREAD

**Issue by eduncan26** [angular-ui/bootstrap#4478]
* Angular - 1.2.28
* ui-bootstrap - 0.11.2

Here is a plunker of a failing scenario: http://plnkr.co/edit/pouPdsf3ugooUiyHMPTV?p=preview

The default template is not accessible via keyboard navigation nor with screen reader assistance. While tabbing through the document you should be able to get keyboard focus on the carousel controls. While using assistive technology, you should also be able to tab-stop on the carousel controls. Neither of these are working as-is.

I understand that newer versions allow for custom templates but there are very minimal changes to the default template that would allow all versions to be much more accessible.

**Comment by wesleycho** [comment]
This is way too old of a version for us to support - for reference, that version was released 1 year ago (minus one day).

**Comment by pkozlowski-opensource** [comment]
@wesleycho the way I understood what @eduncan26 was saying is that we still got this issue...
@eduncan26 did you try the latest release? Can you see the same accessibility issues? A lot of work went into improving this area.

**Comment by eduncan26** [comment]
You have the same accessibility issues with the latest release -- the only thing that really helps is that you support custom templates in 0.13. The default template between 0.11 and 0.13 is essentially the same. I understand not wanting to support 0.11, but the default template should be more accessible than it is currently.

For my project I have updated to 0.13 because I need to make the carousel accessible. I can get you a PR today for improved accessibility just by changing the markup. The accessibility team I work with has also suggested focus path improvements as well (refocus the slide on slide change).

**Comment by wesleycho** [comment]
Do you happen to know what changes are necessary for accessibility? It would help speed things up if you're willing to file a PR with the fixes necessary - otherwise we can try figuring it out.

**Comment by eduncan26** [comment]
I'll get you a PR shortly

**Comment by pkozlowski-opensource** [comment]
> I'll get you a PR shortly

Awesome @eduncan26 ! This is the open source spirit!
